Testscripts/Linux/KDUMP-Config.sh

```diff
@@ -6,7 +6,8 @@ kdump_conf=/etc/kdump.conf
 dump_path=/var/crash
 kdump_sysconfig=/etc/sysconfig/kdump
 boot_filepath=""
-
+#Existed bug for kexec-tools https://bugs.launchpad.net/ubuntu/+source/kexec-tools/+bug/1713940
+kexec_tarball_url="https://mirrors.edge.kernel.org/pub/linux/utils/kernel/kexec/kexec-tools-2.0.18.tar.gz"
```

Comment:
https://mirrors.edge.kernel.org/pub/linux/utils/kernel/kexec/kexec-tools.tar.gz is the latest one, 2.0.19, but it has existed bug